docs/overview/architecture.md

```diff
@@ -1,10 +1,12 @@
-## LDK Architecture
-![LDK Architecture](../assets/ldk-architecture.svg)
+## Architecture
+![Architecture](../assets/architecture.png)
 
-LDK's core components are shown in the center box labeled `lightning`. The boxes
-with dotted borders are LDK's modules — these must be configured with either
-off-the-shelf or custom implementations that you provide.
+Firstly, LDK is not a daemon, but rather its own implementation of the Lightning network written as a SDK and designed to be compiled and executed in your own application. 
 
-`EventHandler` in the diagram is not so much a necessary LDK module, but instead
-refers to the fact that LDK generates events that the user should handle (e.g.
-the `PaymentReceived` event).
+This makes it environment agnostic, allowing you to choose your own data storage, wallet, networking stack and blockchain monitoring. In the diagram above the boxes with dotted borders are LDK's modules — these must be configured with either default or custom implementations that you provide.
+
+We also use an event driven archictecture that that will allow you to act upon number of things asynchronously. For example LDK will generate an event when a payment is received (`PaymentReceived`). More on this in a later section. 
+
+::: tip Langauge Bindings
+
+Although the core SDK is written in Rust, we support a number of other programming langauges. These include C/C++, Java, Kotlin, Swift, JavaScript/Typescript/WASM (Alpha). Check out [examples](../examples.md) to see some of the implementions out in the wild!
```

Comment:
Synced on ldk-dev group regarding the same, also about readiness of bindings. 

"Java/Kotlin, Swift, TypeScript/JavaScript(Beta), C++(Alpha)"

We don't want to list C, because its unusable.

Reply:
Thanks @G8XSU, fixed in - https://github.com/lightningdevkit/lightningdevkit.org/pull/152/commits/b066d58863a207025eb77352c2063b28760e3826